controllers/controllers.js

```diff
@@ -0,0 +1,7 @@
+const { selecTopics } = require("../models/models")
+
+exports.getTopics = (req, res) => {
+    selecTopics().then((topics) => {
+        res.status(200).send({topics})
+    })
```

Comment:
pls add a catch here :) See overview comments for more details re. error handling.